packages/backend/src/open_payments/payment/outgoing/service.test.ts

```diff
@@ -723,12 +729,43 @@ describe('OutgoingPaymentService', (): void => {
             grant
           }
 
+          // Must account for interledger/pay off-by-one issue (even with 0 slippage/fees)
+          const adjustedReceiveAmountValue = debitAmount.value - 1n
+
           for (let i = 0; i < 3; i++) {
             const payment = await outgoingPaymentService.create(options)
             assert.ok(!isOutgoingPaymentError(payment))
+
             expect(payment.grantSpentReceiveAmount?.value ?? 0n).toBe(
-              // Must account for interledger/pay off-by-one issue (even with 0 slippage/fees)
-              BigInt((debitAmount.value - BigInt(1)) * BigInt(i))
+              adjustedReceiveAmountValue * BigInt(i)
+            )
+
+            const spentAmounts = await OutgoingPaymentGrantSpentAmounts.query(
+              knex
+            )
+              .where({ outgoingPaymentId: payment.id })
+              .first()
+            assert(spentAmounts)
+
+            expect(spentAmounts).toEqual(
+              expect.objectContaining({
+                grantId: grant.id,
+                outgoingPaymentId: payment.id,
+                debitAmountCode: debitAmount.assetCode,
+                debitAmountScale: debitAmount.assetScale,
+                paymentDebitAmountValue: debitAmount.value,
+                grantTotalDebitAmountValue: debitAmount.value * BigInt(i + 1),
+                receiveAmountCode: debitAmount.assetCode,
+                receiveAmountScale: debitAmount.assetScale,
+                paymentReceiveAmountValue: adjustedReceiveAmountValue,
+                grantTotalReceiveAmountValue:
+                  adjustedReceiveAmountValue * BigInt(i + 1),
+                intervalDebitAmountValue: null,
+                intervalReceiveAmountValue: null,
+                intervalStart: null,
+                intervalEnd: null,
+                paymentState: 'FUNDING'
+              })
```

Comment:
the grant spent amounts are kind of an implementation detail. at the end of the day its grantSpent[Receive|Debit]Amounts which matter, which is already tested. IDK, left for now but considered not including this.